integration_testing/features/coach/coach-activate-quiz.feature

```diff
@@ -3,29 +3,46 @@ Feature: Coach activates and deactivates quizzes
 
   Background:
     Given I am signed in to Kolibri as coach user
-      And I am on *Coach - '<class>' > Plan > Quizzes > '<quiz>'* page
 
-  Scenario: Change the quiz status to *Active*
-    Given that quiz <quiz> *Status* is *Inactive*
-      When I click the *Options* button
-        And I select *Edit details*
-      Then I see the *Edit quiz details for '<quiz>'* page
-      When I select *Active* under *Status*
-        And I click *Save changes* button
-      Then the page reloads
-#        And I see the snackbar notification *Changes to quiz saved* # No snackbar
-        And I see the quiz *Status* is *Active*
+  Scenario: Change the quiz status to *Open*
+    Given that quiz <quiz> has never been set to the status *Open*
+      When I am on *Coach - '<class>' > Plan > Quizzes > '<quiz>'* page
+      Then I see the *Open Quiz* button on the left sidebar
```

Comment:
Given that *sidebar* appears only on certain browser widths, I'd eliminate that part.
``Then I see the *Open quiz* button``

![quiz-sidebar](https://user-images.githubusercontent.com/1457929/67442978-3cfc1900-f603-11e9-9929-9cf8ff75beb5.png)

Reply:
I notice the icon is `ic_error` next to average score - can this be changed to `ic_info`?